x-pack/test/accessibility/config.ts

```diff
@@ -41,8 +41,11 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       require.resolve('./apps/graph'),
       require.resolve('./apps/security_solution'),
       require.resolve('./apps/ml_embeddables_in_dashboard'),
+      // Please make sure that the remote clusters, snapshot and restore and
```

Comment:
Can we please run this in flaky tester again? I am worried about the sets of tests needing the order? We usually try to make them atomic. Thanks!

Reply:
https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/834